### ISSUE TITLE
Forkify images are undefined in bookmarked + Netlify 

### ISSUE THREAD

**Issue by JannickPepe** [jonasschmedtmann/complete-javascript-course#210]
I have followed a solution about the errors with fraction when running live on Netlify, however I tried the fracty solution but didnt work + the localhost didnt worked either when I tried.

I have also tried to solve the undefined pictures in the bookmarked recipe
[UdemyJS-ForkifySPA-master.zip](https://github.com/jonasschmedtmann/complete-javascript-course/files/7512546/UdemyJS-ForkifySPA-master.zip)
s.

**Comment by JannickPepe** [comment]
@jonasschmedtmann

**Comment by akozdev** [comment]
I set up Forkify with Fracty and latest Parcel. Can you try running it locally and on Netlify?
You can find it here [https://github.com/akozdev/Forkify-Fracty-Parcel2](url).

1. Clone the repository (or download a ZIP)
2. Install dependencies with `npm install`
3. Run `npm start`

**Comment by JannickPepe** [comment]
@akozdev the bookmarked pictures are now there locally, shall add redo the same steps for netlify part... Deploy this git to my own so i can hook it up in netlify process or is there another way :)?

**Comment by akozdev** [comment]
Yeah, you can do that. You can fork that repo and then deploy your fork to Netlify

**Comment by JannickPepe** [comment]
@akozdev the weird part is, when i try to copy your model.js it does not work unless i do this as like in my own code...
export const loadRecipe = async function (id) {
  try {
    const data = await AJAX(`${API_URL}/${id}?key=${KEY}`); 
- where I have the / after API URL so i can load the recipes. However now i can get the bookmarked img on but not the recipe img, if i change image: recipe.image_url to ------> image_url:recipe.image:url then i can get recipe img but the bookmarked img undefined again xD man oh boy LOL

**Comment by JannickPepe** [comment]
is it because of my dependicies ?
"devDependencies": {
    "@parcel/transformer-image": "^2.0.0-rc.0",
    "@parcel/transformer-sass": "^2.0.0-rc.0",
    "parcel": "^2.0.0-rc.0"
  },
  "dependencies": {
    "core-js": "^3.18.2",
    "fractional": "^1.0.0",
    "fracty": "^1.0.9",
    "regenerator-runtime": "^0.13.9"
  }

**Comment by akozdev** [comment]
How does the API_URL look like in your config.js?
Generally, it should look like this
`export const API_URL = 'https://forkify-api.herokuapp.com/api/v2/recipes/';`
with the slash at the end

**Comment by JannickPepe** [comment]
@akozdev  Damn bro it was the motha effin / at the end of recipes in the API_URL. 

only netlify/fracty left XD
this is my json package

{
  "name": "forkify",
  "version": "1.0.0",
  "description": "Recipe application",
  "default": "index.html",
  "scripts": {
    "start": "parcel index.html",
    "build": "parcel build index.html --dist-dir ./dist"
  },
  "author": "jannick pedersen",
  "license": "ISC",
  "devDependencies": {
    "@parcel/transformer-image": "^2.0.0-rc.0",
    "@parcel/transformer-sass": "^2.0.0-rc.0",
    "parcel": "^2.0.0-rc.0"
  },
  "dependencies": {
    "core-js": "^3.18.2",
    "fractional": "^1.0.0",
    "fracty": "^1.0.9",
    "regenerator-runtime": "^0.13.9"
  }
}

**Comment by JannickPepe** [comment]
It yes work now - what a ride,  lol xD

Does Jonas have a Javascript course which has a responsive part / vanilla or with npm :)?

And thanks for your time !

**Comment by akozdev** [comment]
I'm glad it works for you 😀
Jonas has made an HTML/CSS course where he builds a responsive website, but it's not about JavaScript. Mostly about HTML and CSS.

**Comment by JannickPepe** [comment]
@akozdev tbh i am mainly drawn to JS vanilla - i think NPM and frameworks have taken over alot - thats why the founder of NPM created Deno lol xD. Im gonna code Deno maybe in 2023.  If Jonas do a big JS vanilla responsive, ill throw my money xD the adv dom was nice in this course though :) though i prolly gonna look into his CSS and animation course. Need more UX / design experience too on my CV
